setup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+from glob import glob
+import os
+import sys
+
+from setuptools import setup, Extension
+from Cython.Build  import cythonize
+
+if sys.argv[-1] == 'setup.py':
+    print('To install, run \'python setup.py install\'')
+
+if sys.version_info[:2] < (2, 7):
+    print(
+        'networkx-lemon requires Python version 2.7 or later' +
+        ' ({}.{} detected).'.format(*sys.version_info[:2]))
+    sys.exit(-1)
+
+libraries = [
+    ('lemon', {'sources': glob('src/lemon/base.cc') +
+                          glob('src/lemon/bits/*.cc'),
+               'depends': glob('src/lemon/*.h') +
+                          glob('src/lemon/concepts/*.h') +
+                          glob('src/lemon/bits/*.h'),
+               'include_dirs': ['src', 'src/lemon', 'src/lemon/concepts', 
+                                'src/lemon/bits']})]
+
```

Comment:
Well the other parts are sure needed because many header files from `src/lemon` make use of them. However, we may not go with compiling `src/lemon/bits/*.cc` because that seems unnecessary.